src/main/java/com/google/flourbot/entity/EntityModuleImplementation.java

```diff
@@ -1,100 +1,99 @@
 
-        Trigger trigger = optionalTrigger.get();
-        Action action = optionalAction.get();
+    String creatorId = (String) macroMap.get("creatorId");
+    Macro macro = new Macro(creatorId, macroName, trigger, action);
 
-        String creatorId = (String) macroMap.get("creatorId");
-        Macro macro = new Macro(creatorId, macroName, trigger, action);
+    return Optional.of(macro);
+  }
 
-        return Optional.of(macro);
-    }
+  private Optional<Trigger> getTrigger(Map<String, Object> triggerMap) {
 
-    private Optional<Trigger> getTrigger(Map<String, Object> triggerMap) {
-        
-        String triggerType = (String) triggerMap.get("type");
+    String triggerType = (String) triggerMap.get("type");
 
-        switch(triggerType) {
-            case ("Command Trigger"): 
-                String command = (String) triggerMap.get("command");
-                Trigger trigger = new CommandTrigger(command);
+    switch (triggerType) {
+      case ("Command Trigger"):
+        String command = (String) triggerMap.get("command");
+        Trigger trigger = new CommandTrigger(command);
 
-                return Optional.of(trigger);
+        return Optional.of(trigger);
 
-            default:
-                return Optional.empty();
-        }
+      default:
+        return Optional.empty();
     }
+  }
 
-    private Optional<Action> getAction(Map<String, Object> actionMap) {
-        
-        String actionType = (String) actionMap.get("type");
+  private Optional<Action> getAction(Map<String, Object> actionMap) {
 
-        switch(actionType) {
-            case ("Sheet Action"):
-                String[] columnValue = (String[]) actionMap.get("columnValue");
-                String sheetAction = (String) actionMap.get("sheetAction");
-                String sheetUrl = (String) actionMap.get("sheetUrl");
-                Action action = new SheetAppendAction(columnValue, sheetAction, sheetUrl);
+    String actionType = (String) actionMap.get("type");
 
-                return Optional.of(action);
+    switch (actionType) {
+      case ("Sheet Action"):
+        String[] columnValue = (String[]) actionMap.get("columnValue");
+        String sheetAction = (String) actionMap.get("sheetAction");
+        String sheetUrl = (String) actionMap.get("sheetUrl");
+        Action action = new SheetAppendAction(columnValue, sheetAction, sheetUrl);
 
-            default:
-                return Optional.empty();
-        }
-    }
+        return Optional.of(action);
 
+      default:
+        return Optional.empty();
+    }
+  }
 
-    // Singleton Support
-    public static EntityModuleImplementation getInstance() {
-        if (instance == null) {
-            instance = new EntityModuleImplementation();
-        }
-        return instance;
+  // Singleton Support
+  public static EntityModuleImplementation getInstance() {
+    if (instance == null) {
+      instance = new EntityModuleImplementation();
     }
```

Comment:
How is the initializer work going, do you need help with it?

Reply:
I finished initializer in #16